### ISSUE TITLE
How to build on linux without 3pm ?

### ISSUE THREAD

**Issue by stuaxo** [three-x/three-cpp#3]
I have cmake etc and have managed to build previous versions of this in the past, what are the vanilla instructions along the lines of

```
mkdir build
cd build
cmake ...
```

**Comment by elwinarens** [comment]
This project/repo is currently unstable and from a feature perspective equal to the original master.  3pm does nothing more than running cmake (see source) so if you feel comfortable with cmake you should be able to get it running. Big BUT is that i was in the process of decoupling and refactoring big parts of the project and ive left it in an unfinished state. 

Another but is that i'm ending the contract i'm currently working on and i might be designating some time to the project again.

Tl;dr: use the master of the original project for now.

Hope this helps.

**Comment by stuaxo** [comment]
K, I'll try and get the original working again, if I get that far + I'll see what the difference is and put an update here.

**Comment by elwinarens** [comment]
The master branch succesfully builds on travis ci, which builds on a linux box. So it should build on linux. A PR for this project is unnecessary, because it has a different (yet to be finished) setup.

Feel free to ask if you have any issues bootstrapping, but mind that this and the original project is not production ready and imho only worth your time if you're interested in experimenting.
